### PR TITLE
chore(license): unify joint copyright notice in LICENSE file

### LINKED ISSUE #1193: Copyright ownership inconsistencies
When trying to give proper attribution I came up with the following question: Who does this repo's copyright belong to?
The `README` says `Nuxt Community` but the file `LICENSE` still says `Paul Gascou-Vaillancourt`.  
I assume this repo was moved into the nuxt-namespace and the copyright got transferred, so it should say `Nuxt Community` in both places. If this is correct I would appreciate a short confirmation by the maintainer(s) and I'll create a quick PR to make it consistent again.

### PR DESCRIPTION
**Pull request by gekkedev**
resolves #1193 by amending the Nuxt.js copyright

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2021 Nuxt.js Community
```
Comment (divine) [inline LICENSE]:
```suggestion
Copyright (c) Nuxt Community
```

File: README.md [hunk 2]
```diff
@@ -94,4 +94,6 @@ If you have any questions or issues, check out [Discord server](https://discord.
 
 ## License
 
-[MIT License](./LICENSE) - Copyright (c) Nuxt Community
+[MIT License](./LICENSE)  
+Copyright (c) 2021 Nuxt.js Community
```
Comment (divine) [inline README.md]:
```suggestion
Copyright (c) Nuxt Community
```

File: README.md [hunk 3]
```diff
@@ -94,4 +94,6 @@ If you have any questions or issues, check out [Discord server](https://discord.
 
 ## License
 
-[MIT License](./LICENSE) - Copyright (c) Nuxt Community
+[MIT License](./LICENSE)  
+Copyright (c) 2021 Nuxt.js Community  
+Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
```
Comment (divine) [inline README.md]:
```suggestion
```

File: LICENSE [hunk 4]
```diff
@@ -1,5 +1,6 @@
...
 Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
```
Comment (divine) [inline LICENSE:5]:
```suggestion
```

**Review by divine — CHANGES_REQUESTED** [review]
What's the reason for adding a year? Basically, each year maintainers will need to update.

No point to keep Paul in license, his name is already in package.json file.

**Comment by gekkedev** [comment]
> What's the reason for adding a year? Basically, each year maintainers will need to update.
> 
> No point to keep Paul in license, his name is already in package.json file.

There is no use in adding a year indeed, even the whole notice doesn't change the fact that it is copyrighted as it is merely a hint and clarifies the obvious. If you're sure that the previous maintainer's notice can and should be removed, then I'll apply the changes RN. I simply kept both in there in descending order by date because I saw that somewhere in such a format (I believe it was a cryptocurrency fork).

**Comment by rchl** [comment]
I'm not a lawyer but this license specifically mentions that:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

That to me means that everything from the original license should be retained, including the original author.

I'll just ping @paulgv in case he has something to say about that.

**Comment by divine** [comment]
> I'm not a lawyer but this license specifically mentions that:

I know many modules were transferred previously to the nuxt community, and my suggestion was to keep it the same as in other repositories under same organization.

> I'll just ping @paulgv in case he has something to say about that.

My other suggestion would be to add you too to the license and readme file 😉 

Thanks!

**Comment by gekkedev** [comment]
So, we're likely gonna revert the suggestions one by one again? Where would we be without squash merging :slightly_smiling_face:

File: LICENSE [hunk 5]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
+Copyright (c) Nuxt Community
```
Comment (rchl) [inline LICENSE:3]: I guess we could do:
```suggestion
Copyright (c) Nuxt Community
Copyright (c) Rafał Chłodnicki (@rchl)
Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
```

Comment (gekkedev) [reply LICENSE:3]: LGTM will apply that; hope it now represents the history^^

**Review by rchl — APPROVED** [review]

**Comment by rchl** [comment]
Thanks

**Pull request merged**